### PR TITLE
tests: Skip Bigtable samples tests as dependencies are now too old.

### PR DESCRIPTION
**Pull request by amanda-tarafa**
See #992

### CODE REVIEW

File: bigtable/api/SnippetsTest/runTest.ps1 [hunk 1]
```diff
@@ -11,4 +11,6 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations under
 # the License.
-dotnet test --test-adapter-path:. --logger:junit -v n
\ No newline at end of file
+
+    dotnet restore --force
```
Comment (jskeet) [inline bigtable/api/SnippetsTest/runTest.ps1:15]: Is the extra spacing here deliberate, or just copied from the runTests script?

File: bigtable/api/BigtableTest/runTests.ps1 [hunk 2]
```diff
@@ -12,15 +12,17 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
+# Skipping tests because of https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/992
 
-Require-Platform Win*
-Set-TestTimeout 600
+# Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
```
Comment (jskeet) [inline bigtable/api/BigtableTest/runTests.ps1:17]: Is there a way of just returning immediately, so it becomes a two-line change in each direction?

**Review by jskeet — APPROVED** [review]
Wow, we really need to update these tests :)

A couple of suggestions, but fine to merge as-is too.

File: bigtable/api/SnippetsTest/runTest.ps1 (hunk 1 above)
Comment (amanda-tarafa) [reply bigtable/api/SnippetsTest/runTest.ps1:15]: That's just a copy paste thing, fixing ...

**Review by amanda-tarafa — COMMENTED** [review]
I'll push changes shortly and merge if all is green, thanks!

**Pull request merged**